api/SVGElement.json

```diff
@@ -0,0 +1,100 @@
+{
+  "api": {
+    "SVGElement": {
+      "__compat": {
+        "mdn_url": "https://developer.mozilla.org/docs/Web/API/SVGElement",
+        "support": {
+          "webview_android": {
+            "version_added": true
+          },
+          "chrome": {
+            "version_added": true
```

Comment:
There is a note for Chrome : "The properties `offsetParent`, `offsetTop`, `offsetLeft`, `offsetWidth`, and `offsetHeight` are deprecated in Chrome 48."

However I suppose there's an argument for creating these properties as JSON entities with entries such as:

```
 "chrome": {
    "version_added": false,
    "version_removed": "48"
```

or similar.

Reply:
Deprecated doesn't mean removed. 
I think it was removed in version 50. See https://groups.google.com/a/chromium.org/forum/#!msg/blink-dev/JlAEmQpWMWA/63acYbmdDwAJ and https://www.chromestatus.com/features/5724912467574784
I think it would be nice to add them as JSON properties.